docs/install/Knative-custom-install.md

```diff
@@ -207,6 +209,8 @@ files from the Knative repositories:
 | [`monitoring-metrics-prometheus.yaml`][1.4]    | Installs only [Prometheus][2.1]**\***                                                                                          | Serving component                                                 |
 | [`monitoring-tracing-zipkin.yaml`][1.5]        | Installs only [Zipkin][2.3].**\***                                                                                             | Serving component, ELK stack (monitoring-logs-elasticsearch.yaml) |
 | [`monitoring-tracing-zipkin-in-mem.yaml`][1.6] | Installs only [Zipkin in-memory][2.3]**\***                                                                                    | Serving component                                                 |
+| [`monitoring-tracing-jaeger.yaml`][1.7]        | Installs only [Jaeger][2.4].**\***                                                                                             | Serving component, ELK stack (monitoring-logs-elasticsearch.yaml) |
```

Comment:
Please swap the order of these ....-tracing-*.yaml files too (jaeger above zipkin).